mobile/src/main/java/ch/epfl/sdp/mobile/application/StoreDocuments.kt

```diff
@@ -114,6 +115,7 @@ data class TournamentDocument(
     val adminId: String? = null,
     val name: String? = null,
     val maxPlayers: Int? = null,
+    val creationTime: Long? = null,
```

Comment:
Maybe worth specifying the unit there. What would you think of naming this `creationTimeEpochMillis` for instance ?